docs/reference/commandline/dockerd.md

```diff
@@ -1084,6 +1085,31 @@ This setting can also be set per container, using the `--cgroup-parent`
 option on `docker create` and `docker run`, and takes precedence over
 the `--cgroup-parent` option on the daemon.
 
+## Daemon Metrics
+
+The `--metrics-addr` option takes a tcp address to serve the metrics API.
+This feature is still experimental, therefore, the daemon must be running in experimental 
+mode for this feature to work.
+
+To serve the metrics API on localhost:1337 you would specify `--metrics-addr 127.0.0.1:1337`
```

Comment:
I don't know what your general approach to port allocation is, but if you don't have any particular preference I'd suggest grabbing a number off https://github.com/prometheus/prometheus/wiki/Default-port-allocations

This will help your users avoid clashes with other Prometheus exporters.